economics.toml

```diff
@@ -18,18 +18,15 @@
     Denomination = 18 # represents the smallest eGLD subdivision (10^-X eGLD for a denomination of X)
 
 [RewardsSettings]
-    LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
-    DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
+    LeaderPercentage = 0.1 #fraction of value 1 - 10%
+    DeveloperPercentage = 0.3 #fraction of value 1 - 30%
     ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
     ProtocolSustainabilityAddress = "erd1frfpd8sdh02qukg3lt8d8seyxkeznd0lpqukytpq49nrmdme0fnqgd2qt3"
-    TopUpGradientPoint = "3000000000000000000000000" # 3MIL eGLD
-    TopUpFactor = 0.25 # fraction of value 0.25 - 25%
 
 [FeeSettings]
-    MaxGasLimitPerBlock     = "1500000000"
-    MaxGasLimitPerMetaBlock = "15000000000"
-    MinGasPrice             = "1000000000" #will yield min tx fee of 0.00005 eGLD
-    GasPriceModifier        = 0.01
-    MinGasLimit             = "50000"
-    GasPerDataByte          = "1500"
-    DataLimitForBaseCalc    = "10000"
+    MaxGasLimitPerBlock     = "750000000"
```

Comment:
750000000?

Reply:
yes, currently on mainnet we have 750000000